uk.co.compucorp.civicrm.hrcore/gulp/tasks/backstopjs.js

```diff
@@ -94,9 +92,7 @@ function cleanUpAndNotify (success) {
 function constructScenarioUrl (siteUrl, scenarioUrl, usersIds) {
   return scenarioUrl
     .replace('{{siteUrl}}', siteUrl)
-    .replace(/\{\{contactId:([^}]+)\}\}/g, function (fullMatch, user) {
-      return usersIds[user].civi;
-    });
+    .replace(/\{\{contactId:([^}]+)\}\}/g, (__, user) => usersIds[user].civi);
```

Comment:
Would be even better if we can tell exactly what is expected:

.replace(/\{\{contactId:(\w+)\}\}/g, (__, user) => usersIds[user].civi);

Reply:
sorry, i don't understand what you mean

Reply:
You may want to specify exactly what do you expect:

```js
.replace(/\{\{contactId:(\s*\d+)\}\}
// meaning matching only digits and optional spaces
```

But its ok to leave like it is.